core/src/block_sync.rs

```diff
@@ -108,20 +116,43 @@ impl<S: SumeragiTrait, W: WorldTrait> Actor for BlockSynchronizer<S, W> {
     async fn on_start(&mut self, ctx: &mut Context<Self>) {
         self.broker.subscribe::<Message, _>(ctx);
         self.broker.subscribe::<ContinueSync, _>(ctx);
-        ctx.notify_every::<ReceiveUpdates>(self.gossip_period);
+        ctx.notify_every::<PollBlockUpdates>(self.gossip_period);
+        ctx.notify_every::<PollHeightUpdates>(self.gossip_period);
```

Comment:
This should probably be different configurable periods?

Reply:
yes, done